Assets/Scripts/Enemies/EnemyAI.cs

```diff
@@ -13,12 +13,14 @@ public abstract class EnemyAI : MonoBehaviour
     protected GameObject player;
     protected AIPhysics aiPath;
     protected Rigidbody2D rb;
+    protected Animator anim;

```

Comment:
What does anim stand for? Animation or Animator?

Reply:
animator